tests/smoke/environment.bats

```diff
@@ -23,7 +23,67 @@
   l0 environment unlink --bi-directional env_name1 env_name2
 }
 
+@test "environment create --user-data common/user_data.sh test2" {
```

Comment:
Something went wonky while merging I'm guessing? Looks like most of these "new" tests have been deleted or are duplicates.

Reply:
Good catch, yeah I'm looking into that now.